source/MaterialXGenShader/TypeDesc.h

```diff
@@ -156,14 +169,15 @@ class MX_GENSHADER_API TypeDesc
     uint32_t _id;
     uint8_t _basetype;
     uint8_t _semantic;
-    uint16_t _size;
+    uint8_t _size;
+    uint8_t _structIndex;
 };
 
 /// @class TypeDescRegistry
 /// Helper class for type registration.
 class MX_GENSHADER_API TypeDescRegistry
 {
-  public:
+public:
```

Comment:
This spacing change was probably just an automatic reformat by your editor, but I'd recommend restoring the original spacing.